Project.toml

```
@@ -7,6 +7,9 @@ version = "0.1.2"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 
+[compat]
+julia = "1.0, 2"
```

```suggestion
julia = "1.0"
```
I think I've never seen `2` listed here before :p